specification/consumption/resource-manager/readme.md

```diff
@@ -335,4 +354,3 @@ java:
 regenerate-manager: true
 generate-interface: true
 ```
-
```

Comment:
@shbha could you add/update readme file for Java code generation in this PR too? Thanks!